packages/common/src/utils.ts

```diff
@@ -64,3 +65,8 @@ export const parseYAML = (filePath: string, fileContent: string): any => {
     throw error;
   }
 };
+
+export const mergeSchemas = (...schemas: ConfigSchema[]): ConfigSchema => {
+  // Later schemas take precedence in case of key duplication.
```

Comment:
It kinda bugs me that this is implicitly the case, but figured this might be OK for a first iteration.

Honestly I think I'd prefer to have the behaviour such that two conflicting configSchemas (i.e. at the same depth, with the same key) explicitly throws an error - I feel like it'd be less likely to lead to potentially confusing bugs for the user. Would love to get some feedback on the team's thoughts around this.

Reply:
@jamesL92 I planing on opening a new feature for implementing an `extend` keyword for the `.cfgu` file which practicaly need this merge method, with that in mind do you think it is maybe better to put it on the CfguFile Class as a static method?

cc @configu/dev

Reply:
can do, yup :)